routes/Owner/ownerRoutes.js

```diff
@@ -1,6 +1,27 @@
 const express = require('express');
 const router = express.Router();
-//import PropertyOwner from '../../models/PropertyOwner';
+const PropertyOwner = require('../../models/PropertyOwner');
+
+router.post('/login', async (req, res) => {
+    const email = req.body.email;
+    const password = req.body.password;
+    await PropertyOwner.findOne({ email: email, password: password }, (err, propertyOwner) => {
```

Comment:
should  check `activeStatus`  of the propertyOwner (whether its false or true). if its true ONLY  **PropertyOwner** can log in

Reply:
Done.